src/ess/isissans/sans2d.py

```diff
@@ -66,3 +72,25 @@ def mask_detectors(
 
 
 providers = (detector_edge_mask, sample_holder_mask, mask_detectors)
+
+
+def Sans2dWorkflow() -> sciline.Pipeline:
+    """Create Sans2d workflow with default parameters."""
+    from . import providers as isis_providers
+
+    params = default_parameters()
+    sans2d_providers = sans_providers + isis_providers + mantid_providers + providers
+    return sciline.Pipeline(providers=sans2d_providers, params=params)
+
+
+def Sans2dTutorialWorkflow() -> sciline.Pipeline:
+    """
+    Create Sans2d tutorial workflow.
+
+    Equivalent to :func:`Sans2dWorkflow`, but with loaders for tutorial data instead
+    of Mantid-based loaders.
+    """
+    workflow = Sans2dWorkflow()
+    workflow.insert(load_tutorial_run)
+    workflow.insert(load_tutorial_direct_beam)
+    return workflow
```

Comment:
Does it really make sense to hide the insertions in this function? It is convenient for the tutorials but I see 2 issues with it:
- It increases the number of functions that a user needs to know about.
- It is unclear what is different about this tutorial workflow compared to the regular one. Users may assume that there are more fundamental differences than just how files are loaded.

Reply:
- Why does it increase the number? It decreases it from 2 (`load_*`) to 1 (`Sans2dTutorialWorkflow`).
- Why is it unclear? It is in the docstring as well as mentioned in the notebook.

Reply:
> - Why does it increase the number? It decreases it from 2 (load_*) to 1 (Sans2dTutorialWorkflow).

ok

> - Why is it unclear? It is in the docstring as well as mentioned in the notebook (even though that could be extended).

The explanation in the notebook is not completely clear. And many users won't find the docstring. I would like the explanation to be clear about the fact that the workflows are identical except for how they read files and that the files are entirely equivalent and just use a different format.

Reply:
Agreed to add an extra sentence in notebook to address this.